docs/getting_started/faq.rst

```diff
@@ -51,7 +51,12 @@ with dataframes that don't fit into the available memory. As a result, pandas wo
 for prototyping on a few MBs of data do not scale to tens or hundreds of GBs (depending on the size
 of your machine). Modin supports operating on data that does not fit in memory, so that you can comfortably
 work with hundreds of GBs without worrying about substantial slowdown or memory errors. For more information,
-see :doc:`out-of-memory support <getting_started/out_of_core.rst>` for Modin.
+see :doc:`out-of-memory support <getting_started/out_of_core>` for Modin.
+
+How does Modin compare to Dask DataFrame and Koalas?
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Check out this :doc:`page <getting_started/modin_vs_dask_vs_koalas>` detailing the differences!
```

Comment:
Check the link please, I can't navigate to it.

Reply:
Good catch, with the recent refactor there were a few FAQ links broken -- fixed them.